Justfile

```diff
@@ -277,17 +292,13 @@ upload-github-release-assets:
             bin_name="safenode-manager"
             bucket="sn-node-manager"
             ;;
-          sn_faucet)
-            bin_name="faucet"
-            bucket="sn-faucet"
-            ;;
           sn_node_rpc_client)
             bin_name="safenode_rpc_client"
             bucket="sn-node-rpc-client"
             ;;
-          node-launchpad)
-            bin_name="node-launchpad"
-            bucket="node-launchpad"
+          sn_auditor)
+            bin_name="sn_auditor"
```

Comment:
Hey Roland, sorry for the small issue, but the bucket names use hyphens rather than underscores. Could we just keep that consistent please?

Reply:
Aye sure! missed that one, my bad.